source/experimental.html.md.erb

```diff
@@ -3377,13 +3377,13 @@ ScreeningSource
 
 ```
 
-Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause PassFort to consider the check as errored.
+Similar to 'errors' except that the sub_type and data fields are not present and providing warnings will not cause Passfort to consider the check as errored.
```

Comment:
As above, link "errors" to relevant section, format sub_type as code, and change "are not" to "aren't" and "will not" to "won't"